streaming/base/storage/upload.py

```diff
@@ -72,6 +75,7 @@ def get(cls,
                 shard file or remove it after uploading. Defaults to ``False``.
             progress_bar (bool): Display TQDM progress bars for uploading output dataset files to
                 a remote location. Default to ``False``.
+            exist_ok (bool): Throw error if out already exists and not empty. Defaults to ``False``.
```

Comment:
Can you clarify the description for the `exist_ok` arg for the behavior when `True` and `False`? Right now it seems like if I set this to `True` then it will throw error, but that's not actually the case right?